### DISCUSSION TITLE
error for Handle position

### DISCUSSION THREAD

**Discussion by ghost** [xyflow/xyflow#1165] [Q&A] [answered]
Im working on a custom node and constantly getting an error:

Type '"left"' is not assignable to type 'Position'.  TS2322

I took the code straight of the the official Doc.... what am i missing?

**Comment by moklick** [comment, marked as answer]
With TS you need to do: `<Handle position={Position.Left} />` The examples in the repo are written in TS: https://github.com/wbkd/react-flow/tree/main/example/src

**Reply by ghost** [reply]
Thanks!
